src/components/WonderCart.js

```diff
@@ -54,14 +58,18 @@ export function WonderCart(props) {
               </span>
             </div>
             <div className="nfd-flex-none">
-              <Button
-                type="button"
-                variant="upsell"
-                isLoading={showInProgress}
-                onClick={installWonderCart}
-              >
-                {__("Install now", "wp-module-ecommerce")}
-              </Button>
+                <Button
+                  type="button"
+                  as={canAccessGlobalCTB ? "a" : "button"}
+                  data-ctb-id={canAccessGlobalCTB ? "f95ccf1e-3028-4ea7-b2c2-847969348e8b" : null}
+                  data-action={canAccessGlobalCTB ? "load-nfd-ctb" : null}
```

Comment:
Can we remove this line? The `data-action` property isn't needed for global CTBs.